dashboard/app/models/user.rb

```diff
@@ -151,6 +152,7 @@ class User < ApplicationRecord
     progress_table_v2_closed_beta
     lti_roster_sync_enabled
     ai_tutor_access_denied
+    user_provided_us_state
```

Comment:
Please add a JIRA tasks to https://codedotorg.atlassian.net/browse/P20-888 describing that we can remove this attribute post-lockdown. It's safe to do that because this attribute is only used to measure user behavior during the transition period.